### PR TITLE
Fix syntax error on code snippet

### PR DESCRIPTION
**Pull request by Jaxelr**

### CODE REVIEW

File: anti_patterns/deref.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ impl Deref for Bar {
 }
 
 fn main() {
-    let b = Bar { Foo {} };
```
[the remark this answers is not in the document]
Comment (Jaxelr) [reply anti_patterns/deref.md:52]: Added comma as suggested, thanks

[the remark this answers is not in the document]
Comment (MarcoIeni) [reply anti_patterns/deref.md:52]: why the comma? It is removed by cargo fmt. https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=92e09cfb77bcc46dad9f702e2146199a

[the remark this answers is not in the document]
Comment (MarcoIeni) [reply anti_patterns/deref.md:52]: oooooh, you meant the other foo! Fixed :P

**Comment by MarcoIeni** [comment]
Ei Jaxelr, thanks about this.
I formatted the code and remove the `ignore` flag, so that in the future this piece of code will be tested when you run `mdbook test`.
If it looks good for you we merge this 🚀

**Review by MarcoIeni — APPROVED** [review]

**Comment by Jaxelr** [comment]
Looks good to me 👍

**Comment by MarcoIeni** [comment]
Thanks!

**Pull request merged**